app/demo-ibex-super-system/src/main.rs

```diff
@@ -0,0 +1,18 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#![no_std]
+#![no_main]
+
+extern crate panic_halt;
+extern crate riscv_rt;
+
+use riscv_rt::entry;
+
+#[entry]
+fn main() -> ! {
+    const CYCLES_PER_MS: u32 = 0x60_0000;
```

Comment:
Where does this value come from?

Reply:
I just made it up, any values much smaller and it interrupts so fast the core cannot do anything.  I tried to calculate the correct value based on the FPGA clock, but that was absurdly slow.  I think I was missing something about what clock frequency the FPGA was actually running at.

Reply:
Probably add a comment or a bug to follow-up on this if you plan to push this in as is?